src/main/java/com/oltpbenchmark/util/ScriptRunner.java

```diff
@@ -55,7 +55,7 @@ public void runScript(String path) throws IOException, SQLException {
 
         LOG.debug("trying to find file by path {}", path);
 
-        try (InputStream in = this.getClass().getClassLoader().getResourceAsStream(path);
+        try (InputStream in = this.getClass().getResourceAsStream(path);
```

Comment:
What are the implications of this change?

Reply:
I was on a windows machine when doing that PR, and windows resource access works a little bit differently than unix. Using class.getResourceAsStream compare to class.getClassLoader().getResourceAsStream is that path can be relative. I've changed resource path to use absolute path, that way it doesn't change anything, but resolve the small unix/windows quircks.